### ISSUE TITLE
wrong ddns server for ipv6@dynu.com

### ISSUE THREAD

**Issue by mia0x75** [troglobit/inadyn#478]
Right now, inadyn sends the GET request to the dynu's update server, like this

```
GET /nic/update?hostname=subdomain.dynu.com&myip=xxxx:8256:3110:bf40:xxxx:ea2:xxxx:xxxx
```

according to the official document, the correct query parameter should be myipv6 here.

config file:
```
provider ipv6@dynu.com {
    username = 'my-username'
    password = 'my-password'
    hostname = 'subdomain.dynu.com'
    checkip-command = "/sbin/ip -6 addr | grep inet6 | awk -F '[ \t]+|/' '{print $3}' | grep -v ^::1 | grep -v ^fe80"
}
```

**Comment by troglobit** [comment]
Dynu support was added by a 3rd party. I have no means of testing any fixes, so please include relevant documentation/url so we can verify your claims.

Your patch to fix the issue is also welcome, of course.

**Comment by troglobit** [comment]
Fixed in 4aa8af5, PR #479.
